src/dapp-tests/integration/tests.sh

```diff
@@ -49,10 +49,52 @@ dapp_testnet() {
 
   # dynamic fee transaction (EIP-1559)
   seth send "$A_ADDR" "on()" --gas 0xffff --password /dev/null --from "$ACC" --keystore "$TMPDIR"/8545/keystore --prio-fee 2gwei --gas-price 10gwei
+
+  # clean up
+  killall geth
```

Comment:
don't you wanna do the same thing thats in the trap?
https://github.com/dapphub/dapptools/pull/796/files#diff-f18f8b69e1e1cfa0cc72e097dec217d074ca723df9a72f22f7c997e39ceb8100R6

Reply:
Somehow this broke `mktemp` in the subsequent tests. Not sure why. I redid the `dapp testnet` stuff in #797 anyway.